docs/source/algorithms/SetSample-v1.rst

```diff
@@ -29,14 +29,19 @@ respective argument.
 Environment
 ###########
 
-Specifies the sample environment kit to be used. There are two keywords:
+Specifies the sample environment kit to be used. There are two possibilities:
 
-- ``Name``: The name of the predefined kit (required)
-- ``Container``: The id of the container within the predefined kit. (required if there is more than one container defined for the kit).
+Environment Definition File
+---------------------------
```

Comment:
Can the "Environment Definition File" and "Container Geometry and Material" headers be made smaller than the "Environment" header? Think you need to use something other than ----- which is already used for the top level Description header